src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck stopped working.");
```

Comment:
The message in `stopWork` method is not informative enough. It should indicate that the 'Truck' is the type of `Machine` that has stopped working. For example: "Truck has stopped its work."